src/index.css

```diff
@@ -1,13 +1,26 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap');
+
+:root {
+  --color-eggplantPink: #6f455e;
+  /* #7d586e */
+  --color-darkGreen: #456f56;
+  --color-lightGreen: #b5c5bb;
+}
```

Comment:
I think this part is super nice. I have been thinking about creating something like this but never do. So, you are doing great here! Also I like the color combination very much! 😍